doc/source/ray-overview/learn-more.md

```diff
@@ -1,8 +1,25 @@
 # Learn More
 
-Here are some talks, papers, and press coverage involving Ray and its libraries.
+Here are some general rules of thumb, talks, papers, and press coverage involving Ray and its libraries.
 Please raise an issue if any of the below links are broken, or if you'd like to add your own talk!
 
+## Latency numbers every Ray programmer should know
+
+| Stat | Value |
+| --- | --- | 
+| Typical Ray cluster startup time | ~6 minutes |
+| Max size of Ray cluster | ~500 machines |
+| Typical base image size (compressed) | ~4GB | 
```

Comment:
Nit: "base Docker image"

And also question for @yiranwang52 @DmitriGekhtman : is this rule-of-thumb (4GB image) generally true (instead of just true in Anyscale's environment, based on how we build the image)?

Reply:
Updated from `base image` to `base Docker image`. 👍

Reply:
> Nit: "base Docker image"
> 
> 
> 
> And also question for @yiranwang52 @DmitriGekhtman : is this rule-of-thumb (4GB image) generally true (instead of just true in Anyscale's environment, based on how we build the image)?

The numbers are obtainable at the rayproject/ray and rayproject/ray-ml repos on Dockerhub (tags tab)

The vanilla ray image is just over 1GB. The gpu images add 3-4 GB for a total of ~4-5 GB. ML dependencies in ray-ml add another 4GB for a total 8-9
GB.